charmhelpers/core/hookenv.py

```diff
@@ -964,7 +966,16 @@ def charm_dir():
     return os.environ.get('CHARM_DIR')
 
 
+def cmd_exists(cmd):
+    """Return True if the specified cmd exists in the path"""
+    return any(
+        os.access(os.path.join(path, cmd), os.X_OK)
+        for path in os.environ["PATH"].split(os.pathsep)
+    )
```

Comment:
Could use [`shutil.which(cmd)`](https://docs.python.org/3/library/shutil.html?highlight=.which#shutil.which)

Reply:
this doesn't exist in Python 2 so I used something that would work in all cases